CryptoExchange.Net/Sockets/CryptoExchangeWebSocketClient.cs

```diff
@@ -220,6 +223,18 @@ private async Task<CallResult> ConnectInternalAsync()
                     // if _ctsSource was canceled this was already logged
                     _logger.SocketConnectionFailed(Id, e.Message, e);
                 }
+
+                if (e is WebSocketException we)
+                {
+                    // ClientWebSocket.HttpStatusCode is only available in .NET6+ https://learn.microsoft.com/en-us/dotnet/api/system.net.websockets.clientwebsocket.httpstatuscode?view=net-8.0
```

Comment:
Would it be good to use a compiler directive here checking for .NET 6 or higher which uses the new HttpStatusCode property and if lower version fallback to checking for 429?

Reply:
I like that idea